include/proc_ui/procui.h

```diff
@@ -185,14 +186,15 @@ ProcUIInitEx(ProcUISaveCallbackEx saveCallback,
              void *arg);
 
 /**
- * Determines whether the application is running.
+ * Whether the ProcUI library is initialised.
```

Comment:
Nice catch, looks they consider the library as "running" after initialization. 
Maybe this line could be changed to (while keeping the return documentation as is):
```suggestion
 * Determines whether the ProcUI library is running/initialised.
```